src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -0,0 +1,312 @@
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import de.tum.in.www1.artemis.domain.AbstractAuditingEntity;
+import de.tum.in.www1.artemis.domain.Course;
+import de.tum.in.www1.artemis.domain.ExerciseGroup;
+import de.tum.in.www1.artemis.domain.User;
+
+@Entity
+@Table(name = "exam")
+public class Exam extends AbstractAuditingEntity {
+
+    // region CONSTRUCTORS
+    // -----------------------------------------------------------------------------------------------------------------
+    // no arg constructor required for jpa
+    public Exam() {
+    }
+
+    public Exam(Long id, String title, ZonedDateTime visibleDate, ZonedDateTime startDate, ZonedDateTime endDate, String startText, String endText, String confirmationStartText,
+            String confirmationEndText, Integer numberOfExerciseGroups, Course course, Set<ExerciseGroup> exerciseGroups, Set<StudentExam> studentExams,
+            Set<User> registeredUsers) {
+        this.id = id;
+        this.title = title;
+        this.visibleDate = visibleDate;
+        this.startDate = startDate;
+        this.endDate = endDate;
+        this.startText = startText;
+        this.endText = endText;
+        this.confirmationStartText = confirmationStartText;
+        this.confirmationEndText = confirmationEndText;
+        this.numberOfExerciseGroups = numberOfExerciseGroups;
+        this.course = course;
+        this.exerciseGroups = exerciseGroups;
+        this.studentExams = studentExams;
+        this.registeredUsers = registeredUsers;
+    }
+
+    // -----------------------------------------------------------------------------------------------------------------
+    // endregion
+
+    // region BASIC PROPERTIES
+    // -----------------------------------------------------------------------------------------------------------------
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "title", unique = true, nullable = false)
+    private String title;
+
+    /**
+     * student can see the exam in the UI from {@link #visibleDate} date onwards
+     */
+    @Column(name = "visible_date")
+    private ZonedDateTime visibleDate;
+
+    /**
+     * student can start working on exam from {@link #startDate}
+     */
+    @Column(name = "start_date")
+    private ZonedDateTime startDate;
+
+    /**
+     * student can work on exam until {@link #endDate}
+     */
+    @Column(name = "end_date")
+    private ZonedDateTime endDate;
+
+    @Column(name = "start_text")
+    private String startText;
+
+    @Column(name = "end_text")
+    private String endText;
+
+    @Column(name = "confirmation_start_text")
+    private String confirmationStartText;
+
+    @Column(name = "confirmation_end_text")
+    private String confirmationEndText;
+
```

Comment:
please add `maxPoints` as Integer.
Instructors should define this (e.g. 60) and this should be independent of the exercise groups or exercises.
We can use this as additional validation constraint later on, so that we could display a warning if the instructor tries to generate student exams, but the configuration does not fit.

Keep in mind: each student exam should have the same number of points. This is also important when it comes to the calculation of the grades, which we might support at a later stage directly in Artemis